src/tellor_disputables/cli.py

```diff
@@ -125,13 +125,13 @@ async def cli() -> None:
                     Disputable=disputable_strs,
                 )
                 df = pd.DataFrame.from_dict(dataframe_state)
-                print(df.to_markdown(), end="\r")
+                df.to_csv("table.csv")
```

Comment:
You just want to move to logging it to a file? No visual thing in the cli?

Reply:
that's what i was thinking. we might want both, though

Reply:
Up to you. If you think, as a user, that you don't want it printed to the console, then take it out. Otherwise, keep